### PR TITLE
[Breaking] Replace Config.CouchbaseStats by service properties

### PR DESCRIPTION
**Pull request by mcoolive**
- Remove Config.CouchbaseStats
- Add a properties Map to the service configuration object. It is much
more flexible, and it is more relevant.

---
This change is [Reviewable](https://reviewable.io/reviews/criteo/mewpoke/35)

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -23,18 +23,17 @@ service:
...
-     - ep_mem_low_wat
-     - ep_diskqueue_items
-     - ep_oom_errors
-     - ep_tmp_oom_errors
-     - ep_max_size
-  xdcr: # API: replications/XDCR_UUID/MYBUCKET_SOURCE/MYBUCKET_DESTINATION/
-     - bandwidth_usage
-     - percent_completeness
-     - changes_left
+  properties:
```
Comment (gfediere) [inline config.yml]: Please update documentation accordingly

**Comment by erebe** [comment]
-2 for me
Can we stop doing random changes in the config, especially when this is a breaking change.
It may be more flexible but for now we don't need it and it is not motivated by a future upcoming feature.

**Comment by mcoolive** [comment]
In my opinion the actual configuration is not intuitive, and I want to fix it.

The section 'Services' (I think we have to rename it too) is the configuration of the Monitor. If a Monitor need extra parameters (here the list of stats), I expect to find these ones in the same place, not in another section with a name that looks like to my type but different (couchbaseStats <> COUCHBASE_STATS).

**Comment by mcoolive** [comment]
I will split my commit. And I will stack breaking change in a dedicated branch. Breaking change will happen, but we can try to pass them all together.

**Pull request closed without merging**